ci-operator/config/3scale-qe/3scale-deploy/3scale-qe-3scale-deploy-main__3scale-amp-ocp4.13-lp-interop.yaml

```
@@ -0,0 +1,61 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  test-image:
+    name: 3scale-interop-tests
+    namespace: ci
+    tag: v2.13
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: 3scale-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: 3scale-amp-interop-aws
+  cron: 0 1 * * 1
```

```suggestion
  cron: 0 6 * * 1
```
The cron jobs use UTC time, this should translate to the time we usually run out tests.